style.css

```diff
@@ -562,3 +595,222 @@ button:disabled {
   background: #f4f1f0;
   color: #5b5957;
 }
+
+@media screen and (min-width: 1024px) {
+  .menu {
+    display: none;
+  }
+
+  .menu-list {
+    display: flex;
+    justify-content: space-around;
+    width: 293px;
+    align-items: center;
+    font-family: Inter, Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .menu-list img {
+    margin-top: 6px;
+  }
+
+  .intro {
+    background-image: url(./img/Illustration-header.png);
+    height: 870px;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .intro-details p {
+    max-width: 566px;
+  }
+
+  .social-lg {
```

Comment:
- [ ] Add some padding for these social icons at the top, they are currently very close to the other.
![Screenshot from 2021-08-20 04-09-32](https://user-images.githubusercontent.com/65501535/130163294-59e1d2a7-6dad-495f-a423-2f7aed63dcd1.png)